pages/api/applications/index.js

```diff
@@ -0,0 +1,13 @@
+import uploadApplication from '../../../lib/usecases/uploadApplication';
+
+export default async (req, res) => {
+  try {
+    await uploadApplication(req.body);
+    res.statusCode = 201;
+    res.setHeader('Content-Type', 'application/json');
+    res.end();
```

Comment:
Shouldn't return a confirmation id?

Reply:
Good call - looking at it.

Reply:
Done in commit 93a906f02883456185e69c49d23eb4cfb219feaa

Reply:
I think Luka means actually return something like ```const {confirmationId} = await uploadApplication(req.body)```

Reply:
As we discussed, we can change this to just return the ```clientGeneratedId```

Reply:
Yes, I would move the generation of `clientGeneratedId` inside this function using something as `uuid` and returning in case of success as reference.

Reply:
@sirLisko is that what you mean? Check the PR now :)